src/azure-cli-testsdk/azure/cli/testsdk/base.py

```diff
@@ -214,6 +214,9 @@ def __init__(self, method_name):
         self.kwargs = {}
         self.test_resources_count = 0
 
+    def setUp(self):
+        patch_main_exception_handler(self)
```

Comment:
We have to patch `azure.cli.core.util.handle_exception` for `LiveScenarioTest` as well. Otherwise, `AuthenticationError` can't be checked in tests.